README.adoc

```diff
@@ -76,23 +81,26 @@ NAME            URL
 experimental    https://github.com/appsody/stacks/releases/latest/download/experimental-index.yaml
 ----
 
-=== Adding a Kabanero collection hub
+=== Adding a Kabanero Collection Hub
 
 **CLI command:** `appsody repo add <repo-name> <URL>`
 
-As a developer your enterprise architect might provide you with the URL of a collection hub that your organisation
-has defined for developers to use. A collection hub is defined by an `index.yaml` file, which lists all the collections
+As a developer your enterprise architect might provide you with the URL of a Collection Hub that your organisation
+has defined for developers to use. A Collection Hub is defined by an `index.yaml` file, which lists all the collections
 that are available.
 
-To add a collection hub, use the `appsody repo add <repo-name> <URL>` command, supplying a name for the
-repository and the URL for the collection hub. The URL must point to the collection hub's `index.yaml` file.
+To add a Collection Hub, use the `appsody repo add <repo-name> <URL>` command, supplying a name for the
+repository and the URL for the Collection Hub. The URL must point to the Collection Hub's `index.yaml` file.
 
-Run the following command to add the latest public Kabanero Collections repository (it is recommended that the installer of the Kabanero runtime, make a copy of the collection hub and provide that to you, so that you are not suprised when the public collection hub changes)
+In this example, you will add the latest *public* Kabanero Collection Hub repository instead of a cloned Kabanero
+Collection Hub repository that has been customised for your organisation. Run the following command:
 
 ```
 appsody repo add kabanero https://github.com/kabanero-io/collections/releases/latest/download/kabanero-index.yaml
 ```
 
+NOTE: The *public* Kabanero Collection Hub is updated on a regular basis. By using a clone, you avoid adopting unplanned changes.
```

Comment:
The phrase "By using a clone, ..." makes me think that we are using a clone in this case but we're not, this example shows us using the *public* Kabanero Collection Hub so I find that wording confusing.

Reply:
Changed the note and moved it above **Run the following command**

NOTE: The *public* Kabanero Collection Hub is updated on a regular basis. By making a copy of the public hub you avoid adopting unplanned changes.